make/include/wait_for

```diff
@@ -33,7 +33,27 @@ wait_for_job() {
     esac
   done
 
-  until [[ -n "$(kubectl get jobs --namespace "${namespace}" --selector "${selector}" -o jsonpath='{.items[?(@.status.succeeded==1)].status.conditions[?(@.type=="Complete")]}')" ]]; do
+  if [[ -z "${namespace:-}" ]]; then
+    echo "Missing namespace. '-n' must be specified." >&2
+    exit 1
```

Comment:
`return 1`, maybe (to bubble things up to the caller)? Not that it makes a difference…

Reply:
-o errexit wouldn't do an exit 1?

Reply:
No, because this is a function, not a subshell; `return 1` means the caller can catch things, whereas `exit 1` means the shell exits.
(But we probably don't care)

Reply:
I did `return` anyways.